sched/plan_class_spec.h

```diff
@@ -15,9 +15,8 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with BOINC.  If not, see <http://www.gnu.org/licenses/>.
 
-
 // configurable app plan functions.
-// see https://boinc.berkeley.edu/trac/wiki/AppPlanConfig
+// see https://github.com/BOINC/boinc/wiki/AppPlanConfig
```

Comment:
This link is not correct (no such wiki page).
Probably this should be instead https://github.com/BOINC/boinc/wiki/AppPlanSpec ?